README.md

```diff
@@ -1,11 +1,11 @@
 # storage
 
-[![Build Status](https://github.com/aos-dev/go-storage/workflows/Unit%20Test/badge.svg?branch=master)](https://github.com/aos-dev/go-storage/actions?query=workflow%3A%22Unit+Test%22)
-[![Go dev](https://pkg.go.dev/badge/github.com/aos-dev/go-storage/v3)](https://pkg.go.dev/github.com/aos-dev/go-storage/v3)
-[![License](https://img.shields.io/badge/license-apache%20v2-blue.svg)](https://github.com/aos-dev/go-storage/v3/blob/master/LICENSE)
-[![go storage dev](https://img.shields.io/matrix/go-storage:aos.dev.svg?server_fqdn=chat.aos.dev&label=%23go-storage%3Aaos.dev&logo=matrix)](https://matrix.to/#/#go-storage:aos.dev)
+[![Build Status](https://github.com/beyondstorage/go-storage/workflows/Unit%20Test/badge.svg?branch=master)](https://github.com/beyondstorage/go-storage/actions?query=workflow%3A%22Unit+Test%22)
+[![Go dev](https://pkg.go.dev/badge/github.com/beyondstorage/go-storage/v4)](https://pkg.go.dev/github.com/beyondstorage/go-storage/v4)
```

Comment:
Do we plan to release `v4` after this PR merged? Otherwise, we should change our module (or version) after bump version.

Reply:
Out import path has been changed, so we should bump to v4.

Reply:
> Out import path has been changed, so we should bump to v4.

So release new version as long as this PR is merged?

Reply:
Maybe after https://github.com/beyondstorage/specs/issues/74 been implemented.


All those changes will take effect after they finally released, so I think both methods are OK but change them in the same PR makes it easier.

And bump major version first make it easier for go-service-* to do the migration.

Reply:
LGTM